cli.py

```diff
@@ -24,17 +25,15 @@
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
 
-# TODO: this seems kind of weird and heavy on boilerplate -- check it is
-# the best Python can do for us.
-class Options(object):
-    def __init__(self, stack, quiet, verbose, dry_run, local_stack, debug, continue_on_error):
-        self.stack = stack
-        self.quiet = quiet
-        self.verbose = verbose
-        self.dry_run = dry_run
-        self.local_stack = local_stack
-        self.debug = debug
-        self.continue_on_error = continue_on_error
+@dataclass
+class Options:
+    stack: str
+    quiet: bool = False
```

Comment:
Setting defaults here is not needed, just done to match the defaults in the CLI tooling to replicate defaults for repl use.